gitgud/levels/__init__.py

```diff
@@ -1,20 +1,18 @@
-from collections import OrderedDict
-
 from gitgud.levels.intro import all_challenges as intro_challenges
 from gitgud.levels.merging import all_challenges as merging_challenges
 
 from gitgud.levels.util import Level
```

Comment:
You won't need to import this because you'll be importing the levels from the subpackages above